### DISCUSSION TITLE
series 3 table 1

### DISCUSSION THREAD

**Discussion by nadiaBTS** [IBAHCM/RPiR#540] [Registration]
links to the repositories for table 1, for practical series 3
@mhuzzell 
@2394135R 
@nadiaBTS 
@sas-b9 
@morgan-hannah

**Comment by nadiaBTS** [comment]
Nadia Sheikh's: https://github.com/SBOHVM/nadiaBTSSeries03.git

**Comment by mhuzzell** [comment]
Molly Uzzell's repository: https://github.com/SBOHVM/mhuzzellSeries03.git

**Comment by 2394135R** [comment]
Hi! Here is the link to my repository :) 
https://github.com/SBOHVM/lottaruhaSeries03.git

**Comment by morgan-hannah** [comment]
Morgan Hannah's repository: https://github.com/SBOHVM/morganhannahSeries03.git

**Comment by sas-b9** [comment]
Saskia repository: https://github.com/SBOHVM/githubsasb9Series03

**Comment by mhuzzell** [comment]
Here's my repository for series 4: https://github.com/SBOHVM/mhuzzellBCI.git

**Comment by 2394135R** [comment]
Hellloo! Here is the link to my repository for series 4 :)
https://github.com/SBOHVM/lottaruhaBCI.git
- Lotta

**Comment by mhuzzell** [comment]
@nadiaBTS and @sas-b9 could you please add links to your BCI packages, so we can check each other's for practical 4 before the deadline?

(Morgan's isn't linked but it is shared: https://github.com/SBOHVM/morganhannahBCI)

**Reply by sas-b9** [reply]
Hello all done for you, thanks for reminding me!

**Reply by nadiaBTS** [reply]
Sorry Molly for some reason I thought for the 4th practical we didnt have to check each others code! Thanks so much for the reminder, just checked moodle and it does say we have to. I'll upload and check yours now xx

**Comment by nadiaBTS** [comment]
hey, here is my practical 4 repository link: https://github.com/SBOHVM/nadiaBTSBCI
-Nadia
